packages/asset-swapper/test/quote_requestor_test.ts

```diff
@@ -102,9 +114,29 @@ describe('QuoteRequestor', async () => {
                 responseData: unsignedOrder,
                 responseCode: StatusCodes.Success,
             });
+            // A successful response code and good order but for the wrong takerAddress
+            const orderWithNullTaker = testOrderFactory.generateTestSignedOrder({
```

Comment:
If we wanted to get super granular we could actually test to see if a warning was generated for this case ;)